src/System.Private.Uri/tests/FunctionalTests/UriTests.cs

```diff
@@ -892,5 +892,45 @@ public static void Uri_ExtremelyLongScheme_ThrowsUriFormatException()
             string largeString = new string('a', 1_000_000) + ":"; // 2MB is large enough to cause a stack overflow if we stackalloc the scheme buffer.
             Assert.Throws<UriFormatException>(() => new Uri(largeString));
         }
+
+        [Fact]
```

Comment:
All of these tests will need to be attributed to skip on netfx, right?

Reply:
Yep.

Reply:
I actually only had to disable the tests that involve spaces in the port. All of the others have the same behavior.